<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-QHDN499" failures="1" tests="1" name="Default test(failed)" time="13.11" errors="0" timestamp="2024-09-23T17:55:44 IST">
  <testcase classname="practice.test.InvoiceTest" name="createInvoiceTest" time="0.019">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at practice.test.InvoiceTest.createInvoiceTest(InvoiceTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- createInvoiceTest -->
  <testcase classname="practice.test.InvoiceTest" name="createInvoiceWithContactTest" time="0.0">
    <ignored/>
  </testcase> <!-- createInvoiceWithContactTest -->
</testsuite> <!-- Default test(failed) -->
